tests/testthat/test-normalize_SE.R

```diff
@@ -50,4 +50,17 @@ test_that("normalize_SE works as expected", {
   se2 <- normalize_SE(se, "single-agent", nested_confounders = 
                         c(gDRutils::get_SE_identifiers(se, "barcode", simplify = TRUE), "masked"))
   expect_s4_class(se2, "SummarizedExperiment")
+  
+  
```

Comment:
I miss some description what exactly we are testing here (but it may be for another task)